DisplayBundle/BBcode/InternalLinkDefinition.php

```diff
@@ -88,8 +88,10 @@ protected function generateHtml(BBcodeElementNodeInterface $el, $preview = false
             $linkName = $this->nodeManager->getRouteDocumentName($parameters);
             try {
                 $routeCompileParameters = array();
-                if (array_key_exists('contentSearch_contentId', $parameters)) {
-                    $routeCompileParameters['contentId'] = $parameters['contentSearch_contentId'];
+                if (array_key_exists('wildcard', $parameters)) {
+                    foreach ($parameters['wildcard'] as $name => $value) {
+                        $routeCompileParameters[$name] = $value;
```

Comment:
@itkg-nanne : Shouldn't be there a validation of the parameters to check that it contains only allowed characters (no '?' for instance or accents and other disalowed char in url) ?